SPECIFICATION.md

```diff
@@ -121,7 +121,7 @@ LETTER A (U+0041)":
 
 In addition to straight lines, polylines can contain circular arc segments.
 This is done by adding a third parameter, called *bulge*, to the coordinate
-pair of the end coordinate of an arc segment. It specifies the central angle of
+pair of the **start point** of an arc segment. It specifies the central angle of
```

Comment:
Shouldn't we use coordinate consistently? I know it sounds silly here because "coordinate pair of the start coordinate". Can we just say: "to the start coordinate"?

Reply:
Done.